var/spack/repos/builtin/packages/nicstat/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Nicstat(MakefilePackage):
+    """
+    Nicstat is a Solaris and Linux command-line that prints out network
+    statistics for all network interface cards (NICs), including packets,
+    kilobytes per second, average packet sizes and more.
+    """
+
+    homepage = "https://github.com/scotte/nicstat"
+    url      = "https://jaist.dl.sourceforge.net/project/nicstat/nicstat-1.95.tar.gz"
+
+    version('1.95', sha256='c4cc33f8838f4523f27c3d7584eedbe59f4c587f0821612f5ac2201adc18b367')
+
+    def edit(self, spec, prefix):
+        copy('Makefile.Linux', 'makefile')
+        filter_file('CMODEL =\\s+-m32', '', 'makefile')
```

Comment:
You can use `r'...'` for raw formatting so you don't need the double slash:
```suggestion
        filter_file(r'CMODEL =\s+-m32', '', 'makefile')
```

Reply:
Thanks very much for your guide, I fix it in this patch : https://github.com/spack/spack/pull/14173/commits/aacb061bc58bbf686b375c741690d336569274df